src/styles/blocks/nav.scss

```diff
@@ -0,0 +1,47 @@
+.nav {
+  &__list {
+    display: flex;
+  }
+
+  &__item {
+    position: relative;
+
+    margin-right: 20px;
+
+    &:last-child {
+      margin-right: 0;
+    }
+  }
+
+  &__link {
+    display: block;
+    padding: 24px 0;
+
+    font-weight: 500;
+    color: inherit;
+
+    text-decoration: none;
+    text-transform: uppercase;
+
+    @include animation(color, 0.4s);
```

Comment:
let's be consistent about transitions duration and use one duration value for every transition on the page